pkg/constants/metadata.go

```diff
@@ -20,6 +20,9 @@ const (
 	// DevWorkspaceIDLabel is the label key to store workspace identifier
 	DevWorkspaceIDLabel = "controller.devfile.io/devworkspace_id"
 
+	// DevWorkspacePVCTypeLabel is the label key to identify PVCs used by DevWorkspaces and indicate their storage strategy.
+	DevWorkspacePVCTypeLabel = "controller.devfile.io/devworkspace_pvc_type_label"
```

Comment:
I think the actual string value of this label could just be `controller.devfile.io/devworkspace_pvc_type` (i.e. remove the `_label` part at the end), since it's implied by its usage as a label in the PVC spec, and its name `DevWorkspacePVCTypeLabel` in the code.